R/percent_change.R

```diff
@@ -65,12 +70,13 @@ percent_change <- function(x, description = "up", ...) {
         "respectively"
       )
       # Else just crack out two or more percent_change statements
-    } else if (all(!(x > 0.01 | x <= - 0.01))) {
+    } else if (all(!(x > unchanged_limit | x <= - unchanged_limit))) {
       comm <- paste("both", words$same)
```

Comment:
This fails in a very specific niche case; if there are 3 unchanged values it still returns "both" E.g. 
```
upsanddowns::percent_change(c(0.4, 0.5, 0.6),  unchanged_limit = 0.6)
[1] "both unchanged"
```

If it looks like a quick fix, can this be actioned? If not, can it be raised as a new issue and a separate PR? Will need to be fixed in all 3 functions